src/pages/Explore/Explore.tsx

```diff
@@ -27,7 +27,13 @@ class Explore extends React.Component<{}, ExploreState> {
   async loadModules() {
     try {
       const modules = await getPublicModules();
-      this.setState({modules: modules, state: 'success'});
+      const enabledModules = [];
+      for (const i of modules) {
+        if (!i.disabled) {
+          enabledModules.push(i);
+        }
+      }
+      this.setState({modules: enabledModules, state: 'success'});
```

Comment:
Use the built in map/reduce functions in js instead of manually looping.
```suggestion
      this.setState({modules: modules.filter(m => !m.disabled), state: 'success'});
```